packages/lib/src/textarea/Textarea.tsx

```diff
@@ -1,66 +1,178 @@
+  > span {
+    color: ${({ disabled }) => (disabled ? "var(--color-fg-neutral-medium)" : "var(--color-fg-neutral-stronger)")};
+    font-weight: var(--typography-label-regular);
+  }
+`;
+
+const HelperText = styled.span<{ disabled: TextareaPropsType["disabled"] }>`
+  color: ${({ disabled }) => (disabled ? "var(--color-fg-neutral-medium)" : "var(--color-fg-neutral-stronger)")};
+  font-family: var(--typography-font-family);
+  font-size: var(--typography-helper-text-s);
+  font-weight: var(--typography-helper-text-regular);
+  margin-bottom: var(--spacing-gap-xs);
+`;
+
+const Textarea = styled.textarea<{
+  error: TextareaPropsType["error"];
+  verticalGrow: TextareaPropsType["verticalGrow"];
+}>`
+  ${({ verticalGrow }) => {
+    if (verticalGrow === "none") return "resize: none;";
+    else if (verticalGrow === "auto") return `resize: none; overflow: hidden;`;
+    else if (verticalGrow === "manual") return "resize: vertical;";
+    else return `resize: none;`;
+  }};
+  ${scrollbarStyles}
+  padding: var(--spacing-padding-xs) var(--spacing-padding-xs) var(--spacing-padding-xxxs) var(--spacing-padding-xs);
+  background-color: ${({ disabled }) => (disabled ? `var(--color-bg-neutral-lighter)` : `transparent`)};
+  border-radius: var(--border-radius-s);
+  border: ${({ disabled, error }) => (!disabled && error ? "var(--border-width-m)" : "var(--border-width-s)")}
+    var(--border-style-default)
+    ${(props) => {
+      if (props.disabled) return "var(--border-color-neutral-strong)";
+      else if (props.error) return "var(--border-color-error-medium)";
+      else if (props.readOnly) return "var(--border-color-neutral-strong)";
+      else return "var(--border-color-neutral-dark)";
+    }};
+
+  ${(props) =>
+    !props.disabled
+      ? `&:hover {
+        border-color: ${
+          props.error
+            ? "var(--border-color-error-strong)"
+            : props.readOnly
+              ? "var(--border-color-neutral-stronger)"
+              : "var(--border-color-primary-strong)"
+        };
+      }
+      &:focus, &:focus-within {
+        border-color: transparent;
+        outline-offset: -2px;
+        outline: var(--border-width-m) var(--border-style-default) var(--border-color-secondary-medium);
+      }`
+      : "cursor: not-allowed;"};
+
+  color: ${({ disabled }) => (disabled ? "var(--color-fg-neutral-medium)" : "var(--color-fg-neutral-dark)")};
+  font-family: var(--typography-font-family);
+  font-size: var(--typography-label-m);
+  font-weight: var(--typography-label-regular);
+  ::placeholder {
+    color: ${({ disabled }) => (disabled ? "var(--color-fg-neutral-medium)" : "var(--color-fg-neutral-strong)")};
+  }
+`;
+
+const ErrorMessage = styled.span`
+  display: flex;
+  align-items: center;
+  color: var(--color-fg-error-medium);
+  font-family: var(--typography-font-family);
+  font-size: var(--typography-helper-text-s);
+  font-weight: var(--typography-helper-text-regular);
+  margin-top: var(--spacing-gap-xs);
```

Comment:
A gap of var(--spacing-gap-xs) is missing between the icon and the error string.

Reply:
Fixed!